spec/bundler/fetcher/compact_index_spec.rb

```diff
@@ -57,6 +54,22 @@ def remove_cached_md5_availability
 
         after { remove_cached_md5_availability }
 
+        context "when FIPS-mode is active" do
+          before do
+            allow(OpenSSL).to receive(:fips_mode).and_return(true)
+          end
+
+          it "returns false" do
+            expect(compact_index).to_not be_available
+          end
+        end
+
+        it "returns true" do
+          expect(compact_index).to be_available
+        end
+      end
+
+      context "when OpenSSL is FIPS-enabled", :ruby => "< 2.0.0" do
```

Comment:
I'm guessing that Ruby 1.8.7 in Travis has a version of OpenSSL that has `fips_mode` available, so the `:ruby` tag is not necessary. But we should try and have a test that covers when `fips_mode` is not available.